workbench/icons/Makefile.am

```diff
@@ -0,0 +1,20 @@
+icondir = $(datadir)/icons/hicolor/16x16/apps
```

Comment:
I see an icon, just the wrong size. It seems to be missing in ```/usr/share/icons/elementary-xfce/mimes/16```. So it clearly looks like a theme problem.